CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 1.0.0
```

Comment:
cc @kevmoo as he likes things moving to 1.0.0

Reply:
...a bit 🕺